app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -152,5 +153,50 @@
         });
       }
     }
+
+    /*
+    * Check that the current item is infusable
+    * Only legendary or Exotic items are infusable.
+    */
+    vm.isInfusable = function isInfusable(item) {
+      // Infuse perk's id is 1270552711
+      return _.contains(item.talentPerks, 1270552711);
+    }
+
+    /*
+    * Show the icon for infusion, weapon or armor
+    */
+    vm.icon = function icon(item) {
+      var icon = "";
+      if (item.sort && item.sort === "Weapons") {
```

Comment:
Syntactically...I don't like multiple returns, the last example with the ternary operator is fine...but is less readable to newcomer.
But if we chose a convention for this cases it's ok, maybe better...